src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -43,15 +51,34 @@ const AppealsV2StatusPage = ({ appeal, fullName }) => {
     !hideDocketStatusTypes.includes(status.type) &&
     !hideDocketAppealTypes.includes(appealType);
 
+  const filteredAlerts = alerts.filter(a => a.type !== ALERT_TYPES.cavcOption);
+  const afterNextAlerts = (
+    <div>
+      {alerts
+        .filter(a => a.type === ALERT_TYPES.cavcOption)
```

Comment:
This is great as is. Should we also think about doing something like Ruby's `Enumerable#partition` so we only need to filter this once, as part of a refactor later on?

Reply:
I'm not familiar with this...what do you mean?

Reply:
It looks like there is a [lodash partition function](https://lodash.com/docs/4.17.5#partition) available as well, actually, in case that is of interest to use. Using these we could produce both of the arrays needed in one fell swoop.

Reply:
Ahh, I see. Yeah, that would make sense. In this case, I think I'm going to avoid the lodash dependency. The performance gain is minimal since this array is only ever going to be like 5 items long at most (the number of saved iterations on `O(2n)` vs `O(n)` as `n` approaches 0 is tiny).

I'll definitely keep an eye out for use cases for this where we already use lodash, though.

TIL: `_.partition()` :+1: